tutorials/notebooks/pytorch/pruning/example_pytorch_pruning_mnist.ipynb

```diff
@@ -0,0 +1,412 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Structured Pruning of a Fully-Connected PyTorch Model\n",
+    "\n",
+    "Welcome to this tutorial, where we will guide you through the process of training, pruning, and retraining a fully connected neural network model using the PyTorch framework. The tutorial is organized in the following sections: \n",
```

Comment:
Please add a google colab link for this tutorial.

Reply:
I need the file to be pushed first or else the test fails on missing link